ui/StatusQ/src/StatusQ/Popups/StatusMenuSeparator.qml

```diff
@@ -4,7 +4,7 @@ import QtQuick.Controls 2.13
 import StatusQ.Core.Theme 0.1
 
 MenuSeparator {
-    height: visible && enabled ? implicitHeight : 0
+    implicitHeight: visible && enabled ? implicitHeight : 0
```

Comment:
Why this change?

Reply:
ah, because is used in Layout type

Reply:
although now that I see it probably it's not necessary

Reply:
removed! also fixed id verification popup 
<img width="617" alt="pp" src="https://user-images.githubusercontent.com/31625338/218525555-c5786d3d-dd7b-4c96-ae07-1de74f5115ee.png">